src/PrestaShopBundle/Install/AbstractInstall.php

```diff
@@ -54,7 +54,7 @@ public function setError($errors)
             $errors = array($errors);
         }
 
-        $this->errors[] = $errors;
+        $this->errors += $errors;
```

Comment:
Watch out, the `+` operation doesn't merge duplicate keys: https://stackoverflow.com/questions/7059721/array-merge-versus

Example: https://3v4l.org/9R361
```php
$a = [0 => '"0" in A', 'sameKey' => 'value from A', 'onlyInA' => 'only in A'];
$b = [0 => '"0" in B', 'sameKey' => 'value from B', 'onlyInB' => 'only in B'];

print_r($a + $b);
print_r(array_merge($a, $b));
```
```
Array
(
    [0] => "0" in A
    [sameKey] => value from A
    [onlyInA] => only in A
    [onlyInB] => only in B
)
Array
(
    [0] => "0" in A
    [sameKey] => value from B
    [onlyInA] => only in A
    [1] => "0" in B
    [onlyInB] => only in B
)
```
Unless you're specifically targeting this behavior, it's best to pick between:
- use `array_merge` if you want to append
- use `=` if you want to replace

Reply:
True, I fixed that point on the autoupgrade, but I forgot the previous PRs.